packages/dice-webapp/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
+import React, { useState } from "react";
+import Image from "next/image";
+import LogoPath from "@/public/DICE.svg";
+import { IoMenuOutline } from "react-icons/io5";
+import { FaDiceD20 } from "react-icons/fa";
+import { usePathname } from "next/navigation";
+import { cn } from "@nextui-org/system";
+import { signOut } from "next-auth/react";
+import { Button } from "@nextui-org/button";
+import { IoMdLogOut } from "react-icons/io";
+import { orgSidebarData } from "./sidebar-data";
+import Link from "next/link";
+
+function Hover(
+  props: React.HtmlHTMLAttributes<HTMLDivElement> & { isActive?: boolean },
+) {
+  return (
+    <div
+      {...props}
+      className={cn(
+        "py-2 px-3 hover:bg-gray-200 dark:hover:bg-zinc-900  transition-all cursor-pointer rounded-sm",
+        {
+          "dark:border-b-1 dark:border-b-white border-b-1 border-b-black rounded-sm":
+            props.isActive,
+        },
+        props.className,
+      )}
+    />
+  );
+}
+
+type Props = {};
+
+export default function Sidebar({}: Props) {
+  const [isSidebarOpen, setSidebarOpen] = useState(false);
+  function toggleSidebar() {
+    setSidebarOpen(!isSidebarOpen);
+  }
+  return (
+    <div
+      className={cn(
+        "  h-dvh w-[full] flex-col  flex text-slate-50 dark:border-r-1 dark:border-r-white border-r-1 border-r-black  ",
+      )}
+    >
+      <section className="flex items-center gap-4 p-8">
+        <Hover
+          className={cn("rounded-full py-2 px-2", {
+            "mx-2": !isSidebarOpen,
+          })}
+          onClick={toggleSidebar}
+        >
+          <IoMenuOutline
+            className={cn("text-3xl text-black dark:text-white", {
+              "text-4xl": !isSidebarOpen,
+            })}
+          />
+        </Hover>
+        <div
+          className={cn("text-3xl text-black dark:text-white", {
+            "visibility: hidden": !isSidebarOpen,
+          })}
+        >
+          <FaDiceD20 />
+        </div>
+        <Image
+          className={cn("dark:invert", {
+            "visibility: hidden": !isSidebarOpen,
+          })}
+          src={LogoPath}
+          alt="DICE"
+        />
+      </section>
+      <main
+        className={cn("p-8 flex  flex-col items-start w-[full]  h-full", {
+          "w-[full] ": !isSidebarOpen,
+        })}
+      >
+        {orgSidebarData.map((d, i) => (
+          <>
+            <SidebarItem
+              activeIcon={d.activeIcon}
+              defaultIcon={d.defaultIcon}
+              isSidebarOpen={isSidebarOpen}
+              path={d.path}
+              title={d.title}
+            />
+          </>
+        ))}
```

Comment:
Give meaningful names to item, remove the unwanted wrapper `<></>` as suggestion below
```suggestion
        {orgSidebarData.map((d, i) => (
            <SidebarItem
              activeIcon={item.activeIcon}
              defaultIcon={item.defaultIcon}
              isSidebarOpen={isSidebarOpen}
              path={item.path}
              title={item.title}
            />
        ))}
```